.github/workflows/build_and_test.yml

```diff
@@ -43,13 +43,30 @@ jobs:
       - name: make
         run: make clippy build test
 
+      - name: Add aarch64 Target with Rustup
+        if: runner.os == 'Linux'
+        run: rustup target add aarch64-unknown-linux-gnu
+        
+      - name: Install Cross Tool
+        if: runner.os == 'Linux'
+        run: cargo install cross
+
+        # Cross build and test for aarch64-unknown-linux-gnu
+      - name: cross Build for aarch64-unknown-linux-gnu
+        if: runner.os == 'Linux'
+        run: make cross-build
+
+      - name: cross test for aarch64-unknown-linux-gnu
+        if: runner.os == 'Linux'
+        run: make cross-test
+
       # lcov on ubuntu-latest (22.04) is still version 1.14 that doesn't have the "--erase-functions" option we want
       - name: clean up coverage Linux
         if: runner.os == 'Linux'
         run: |
           grcov . --binary-path target/debug/ -s . -t lcov --branch --ignore-not-existing --ignore "/*" -o lcov.info
           lcov --remove lcov.info -o lcov.info
-          
+        
```

Comment:
Need to rebase? This should do coverage for Linux too

Reply:
i had rebased ur branch